components/brave_search_conversion/p3a.h

```diff
@@ -22,6 +22,15 @@ inline constexpr char kSearchPromoBannerCHistogramName[] =
     "Brave.Search.Promo.BannerC";
 inline constexpr char kSearchPromoBannerDHistogramName[] =
     "Brave.Search.Promo.BannerD";
+inline constexpr char kSearchPromoDDGBannerAHistogramName[] =
+    "Brave.Search.Promo.DDGBannerA";
+inline constexpr char kSearchPromoDDGBannerBHistogramName[] =
+    "Brave.Search.Promo.DDGBannerB";
+inline constexpr char kSearchPromoDDGBannerCHistogramName[] =
+    "Brave.Search.Promo.DDGBannerC";
+inline constexpr char kSearchPromoDDGBannerDHistogramName[] =
+    "Brave.Search.Promo.DDGBannerD";
+
```

Comment:
everything looks good, we just need to add these to the metric list here: https://github.com/brave/brave-core/blob/374231672c477bb57a71225b748fb3ddd430b9f0/components/p3a/metric_names.h#L26-L28

Reply:
Thanks updated - https://github.com/brave/brave-core/pull/23889/commits/b96c033bd6d2d004b8815d36a987b5010ad5733c